### PR TITLE
Better skeleton boilerplate

### PR DESCRIPTION
**Pull request by lucalanca**

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -1,15 +1,24 @@
...
+  // we are using a custom transformation that converts the entryGlobs on config
+  // to a valid webpack entries object by using glob.sync.
   entry: config.entryGlobs.scripts
-    .map(globEntries)
-    .reduce((a, b) => Object.assign({}, a, b), {}),
+    .map(globPath => glob.sync(globPath))
+    // flatten [[], [], ...] to []
+    .reduce((a, b) => a.concat(b), [])
+    // convert array of files to Object of {chunkName: chunkPath}
+    .reduce((a, currentEntryPath) => {
```
Comment (lejoe) [inline webpack.config.js]: Any way we can avoid the hardcoded `src/` ?

File: src/layouts/default/style.scss [hunk 2]
```diff
@@ -0,0 +1,5 @@
+@import "sensible/mediaqueries";
```
Comment (lejoe) [inline src/layouts/default/style.scss]: What about moving that to the breakpoint material?

File: src/pages/index.pug [hunk 3]
```diff
@@ -1,7 +1,8 @@
-extends ../layouts/default
+extends ../layouts/default/layout
```
Comment (lejoe) [inline src/pages/index.pug]: ➿

File: src/layouts/default/style.scss (hunk 2 above)
Comment (lucalanca) [inline src/layouts/default/style.scss]: good point 👍

File: src/pages/index.pug (hunk 3 above)
Comment (lucalanca) [inline src/pages/index.pug]: do you have an alternative. Maybe renaming to `template.pug`?

File: webpack.config.js (hunk 1 above)
Comment (lucalanca) [inline webpack.config.js]: how, yes. We can use a config variable. 🗡

Comment (lucalanca) [inline webpack.config.js]: done,

File: src/layouts/default/style.scss (hunk 2 above)
Comment (lucalanca) [inline src/layouts/default/style.scss]: done.

File: src/layouts/default/layout.pug [hunk 4]
```diff
@@ -6,7 +6,7 @@ html(lang=language)
     meta(name='viewport', content='width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no')
     meta(name='format-detection', content='telephone=no')
 
-    link(rel='stylesheet', href=relativePath + 'css/styles.css')
+    link(rel='stylesheet', href=relativePath + 'css/layouts/default/style.css')
```
Comment (lejoe) [inline src/layouts/default/layout.pug]: Not sure this is the simpliest output. But at least it's self explaining so let's keep it for now

**Comment by lejoe** [comment]
Two last things:
- We [wanted to rename `modules` with `components`](https://docs.google.com/document/d/1YAeTq64euQbjp07JIuLVUmD2cQBj7okGRzxAYxWd6p8/edit#)
- In the base scss there's a .about class. Let's remove it at the same time

**Comment by lucalanca** [comment]
@lejoe Done and done.

File: src/pages/index.pug (hunk 3 above)
Comment (lejoe) [inline src/pages/index.pug]: nah. fine for now.

**Pull request merged**